src/jira.js

```diff
@@ -753,6 +753,9 @@ export default class JiraApi {
    * @param {integer} [startAt=0] - The index of the first user to return (0-based)
    * @param {integer} [maxResults=50] - The maximum number of users to return (defaults to 50).
    */
+  /**
+ * @deprecated
+ */
```

Comment:
Hey sorry, this should be in the same jsdoc

```suggestion
   * @deprecated
   */
```

Reply:
@Seth10001 ah, okay thanks - done.